### DISCUSSION TITLE
monitor死循环检测问题

### DISCUSSION THREAD

**Discussion by sea520-coder** [cloudwu/skynet#1190] [Q&A]
skynet_monitor是只能检查一个消息的处理超时或死循环？
例子: A处理后发消息给B，B处理后发消息给A，A再发消息给B，这样的死循环能检查到吗？
有人能帮忙解惑吗

**Comment by Edimier** [comment]
业务上的死循环，有个skynet_monitor给你兜个底知道自己写了死循环就不错了，想要服务间检测消息发送的死循环检测，最好还是写业务的时候去避免
